specify_pose/proposal.md

```diff
@@ -188,19 +161,20 @@ It's unclear which one may be better. In ROS, `rotation` is used for a
 transform, while `orientation` is used for a pose. However, they both appear
 equivalent.
 
-#### 1.1 (A) Values for `//pose/rotation/@type` or (B) (`@rotation_type`)
+#### 1.1  Values for `//pose/@rotation_format` and `//pose/@degrees` 
 
-The values of `@type` (or `@rotation_type`) that are permitted:
+The permutations of `@rotation_format` and `@degrees` that are
+permitted:
 
-* `rpy_degrees` - A 3-tuple representing Roll-Pitch-Yaw in degrees, which maps
-to a rotation as specified here.
+* `@rotation_format="rpy"`, `@degrees="true"` - A 3-tuple representing
+  Roll-Pitch-Yaw in degrees, which maps to a rotation as specified here.
     * This should be used when the rotation should generally be human-readable.
-* `rpy_radians` - Same as `rpy_degrees`, but with radians as the units for each
+* `@rotation_format="rpy"`, `@degrees="false"` - Same as above, but with radians as the units for each
 angle. This is provided for legacy purposes and ease of conversion.
```

Comment:
Saying that this is provided for legacy purposes suggests that this may be deprecated in the future. Is that the case? Aren't there valid use cases for radian RPY?

Reply:
Changed to "This is the default for legacy purposes" in https://github.com/ignitionrobotics/sdf_tutorials/pull/56/commits/a2584253cdc44b7ded902958ce352c49fe3c04a4